nuxeo-coldstorage-web/elements/nuxeo-coldstorage-infobar.js

```diff
@@ -70,12 +70,43 @@ class ColdStorageInfobar extends mixinBehaviors([FiltersBehavior, FormatBehavior
        * Input document.
        */
       document: Object,
+      /**
+       * Coldstorage status of the file.
+       */
+      _documentStatus: {
+        type: String,
+        computed: '_getDocumentStatus(document)',
+      },
+      /**
+       * Infobar message.
+       */
+      _title: {
+        type: String,
+        computed: '_getTitle(document)',
+      },
     };
   }
 
   _contentStoredInColdStorage(doc) {
     return this.hasFacet(doc, 'ColdStorage') && doc.properties && doc.properties['coldstorage:coldContent'];
   }
+
+  _getDocumentStatus(document) {
+    if (document.properties['coldstorage:beingRetrieved']) {
+      return 'beingRetrieved';
+    }
+    if (document.properties['coldstorage:downloadableUntil']) {
+      return 'retrieved';
+    }
+    return 'availableToBeRetrieved';
```

Comment:
I am wondering if we can rename the third status `availableToBeRetrieved` as now we can do a Restore. We use for example `contentNotAvailable` or `storedInColdStorage`